### ISSUE TITLE
Add an unentitled way to identify if an account be controlled by account caps

### ISSUE THREAD

**Issue by btspoony** [onflow/cadence#3312]
### Issue to be solved

The current exposed interface of the account cannot allow anyone to know whether the address has an Account Capability.
This will make it impossible to implement some logic that needs to determine the controllability of the address.

Discussion context: https://discord.com/channels/613813861610684416/1235590034837475378

### Suggested Solution

Add a method that can be used by any unentitled account to determine the existence of an Account Capability.

**Comment by turbolent** [comment]
Even when being able to determine there are no account capability controllers on an account, a contract deployed to the account may expose similar functionality (e.g. allows adding keys).

**Comment by btspoony** [comment]
> Even when being able to determine there are no account capability controllers on an account, a contract deployed to the account may expose similar functionality (e.g. allows adding keys).

Yes, but currently there is no way to determine the uncontrollability.

Once this is available, combined with the judgment of not deploying any contracts, it can be determined that the address is permanently uncontrollable.
